exchanges/kraken/kraken_test.go

```diff
@@ -2261,3 +2262,43 @@ func TestIsPerpetualFutureCurrency(t *testing.T) {
 		t.Error("expected true")
 	}
 }
+
+func TestGetOpenInterest(t *testing.T) {
+	t.Parallel()
+	_, err := k.GetOpenInterest(context.Background(), key.PairAsset{
+		Base:  currency.ETH.Item,
+		Quote: currency.USDT.Item,
+		Asset: asset.USDTMarginedFutures,
+	})
+	assert.ErrorIs(t, err, asset.ErrNotSupported)
+
+	cp1 := currency.NewPair(currency.PF, currency.NewCode("ETHUSD"))
+	cp2 := currency.NewPair(currency.PF, currency.NewCode("XBTUSD"))
+	sharedtestvalues.SetupCurrencyPairsForExchangeAsset(t, k, asset.Futures, cp1, cp2)
+
+	resp, err := k.GetOpenInterest(context.Background(), key.PairAsset{
+		Base:  cp1.Base.Item,
+		Quote: cp1.Quote.Item,
+		Asset: asset.Futures,
+	})
+	assert.NoError(t, err)
+	assert.NotEmpty(t, resp)
+
+	resp, err = k.GetOpenInterest(context.Background(),
+		key.PairAsset{
+			Base:  cp1.Base.Item,
+			Quote: cp1.Quote.Item,
+			Asset: asset.Futures,
+		},
+		key.PairAsset{
+			Base:  cp2.Base.Item,
+			Quote: cp2.Quote.Item,
+			Asset: asset.Futures,
+		})
+	assert.NoError(t, err)
+	assert.NotEmpty(t, resp)
+
+	_, err = k.GetOpenInterest(context.Background())
+	assert.NoError(t, err)
+	assert.NotEmpty(t, resp)
```

Comment:
@gloriousCode Empty test.
Suspect that `_` needed to be `resp` on 2301

Reply:
You were too late! I've added that fix on https://github.com/thrasher-corp/gocryptotrader/pull/1439 for now